### ISSUE TITLE
The `prefer-add-event-listener` rule should not autofix event clearing

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/eslint-plugin-unicorn#163]
These should be reported, but not auto-fixed:

```js
foo.onclick = undefined;
```

```js
foo.onclick = null;
```

These need to be handled manually, as `removeEventListener` requires the listener function be able to unsubscribe.

**Comment by sindresorhus** [comment]
// @johnwu93

**Comment by IssuehuntBot** [comment]
@issuehuntfest has funded $40.00 to this issue. [See it on IssueHunt](https://issuehunt.io/repos/55832243/issues/163)

**Comment by medusalix** [comment]
If you don't mind, I'll be working on this issue now as @johnwu93 seems to be inactive.

**Comment by sindresorhus** [comment]
@medusalix Go ahead :)

**Comment by IssuehuntBot** [comment]
@medusalix has submitted a pull request. [See it on IssueHunt](https://issuehunt.io/repos/55832243/issues/163)

**Comment by IssuehuntBot** [comment]
@sindresorhus has rewarded $36.00 to @medusalix. [See it on IssueHunt](https://issuehunt.io/repos/55832243/issues/163)

- :moneybag: Total deposit: $40.00
- :tada: Repository reward(0%): $0.00
- :wrench: Service fee(10%): $4.00
